e2e/README.md

```diff
@@ -38,6 +38,31 @@ You may also run the test command with custom arguments yourself (assuming you'v
 go test -timeout 90m -v -run Test_All ./
 ```
 
+## Running locally in VS code debug mode
+You can also run the test locally in VS code debug mode by adding the required test env variables to settings.json.
+
+Steps:
+1. Go to Settings (click gear button at the bottom left corner or press Ctrl+, (windows))
```

Comment:
nit: can you format the Ctrl+, like this `Ctrl`+`,`? Otherwise it might be unclear whether the comma is part of the sentence of the command